README.md

```diff
@@ -1,14 +1,14 @@
 # Extensions catalog
 
-This is a placeholder for definitions of AsyncAPI specification extensions and bindings objects.
+This is a placeholder for definitions of AsyncAPI specification extensions.
 
 #### Index:
 
 <!-- TOC depthFrom:2 depthTo:2 -->
 
 - [What's an extension?](#whats-an-extension)
-- [Examples of extension definition](#examples-of-extension-definition)
-- [Types of extensions](#types-of-extensions)
+- [Usage in an AsyncAPI document](#usage-in-an-AsyncAPI-document)
+- [Twitter Extension](./extensions/twitter/)
```

Comment:
I don't think it should be here in main TOC

better add new section in readme, after "what's an extension" called `List of extensions` and there all extensions should be added to the list. Although I think it will be hard to maintain and better to really just have a sentence in "what's an extension" section that says something like "all the extensions from the catalog are in `extensions` directory" (and make it clickable link to extensions dir)